node/pkg/fetcher/collector.go

```diff
@@ -124,17 +123,15 @@ func calculateAggregatedPrice(valueWeightedAveragePrice, medianPrice float64) fl
 	return valueWeightedAveragePrice*(1-DefaultMedianRatio) + medianPrice*DefaultMedianRatio
 }
 
-func insertAggregateData(ctx context.Context, id int32, aggregated float64) error {
-	if aggregated == 0 {
-		return nil
-	}
-	err1 := insertLocalAggregateRdb(ctx, id, aggregated)
-	err2 := insertLocalAggregatePgsql(ctx, id, aggregated)
-
-	if err1 != nil || err2 != nil {
-		return fmt.Errorf("errors occurred in insertAggregateData: %v, %v", err1, err2)
+func (c *Collector) insertAggregateData(ctx context.Context, aggregated float64) error {
```

Comment:
how about `streamLocalAggregate` for this function name? it doesn't directly stores the data but streams the data to the channel